libs/sdk-ui-tests-e2e/cypress/tools/chart.ts

```diff
@@ -140,25 +140,42 @@ export class Chart {
         return this;
     }
 
-    private getTooltipContents(index: number) {
-        cy.get(
-            `.highcharts-series-${index}.highcharts-tracker rect, .highcharts-series-${index}.highcharts-tracker path `,
-        )
-            .first()
-            .trigger("mouseover", { force: true });
-
-        cy.wait(500);
+    getHighChartSeriesTooltip(index: number, pointIndex = 0) {
```

Comment:
these actions do not get the tooltip, just hover on chart. please  change method name

Reply:
Name changed to hoverOnHighChartSeriesTooltip()